.github/workflows/npm.yml

```diff
@@ -64,6 +66,6 @@ jobs:
 
       - name: Publish
         working-directory: ${{ env.DESTINATION_PATH }}
-        run: npm publish --tag ${{ inputs.tag }}
```

Comment:
@alfetopito , I know I added this somewhere. This made it work

Reply:
Hum, it defaults to `restricted`

> [--access <public|restricted>] Tells the registry whether this package should be published as public or restricted. Only applies to scoped packages, which default to restricted. If you don't have a paid account, you must publish with --access public to publish scoped packages.

https://docs.npmjs.com/cli/v6/commands/npm-publish